tests/src/FunctionalJavascript/SettingsFormTest.php

```diff
@@ -9,7 +9,7 @@
 /**
  * Tests the corporate contact forms.
  */
-class CorporateContactFormTest extends WebDriverTestBase {
+class SettingsFormTest extends WebDriverTestBase {
```

Comment:
This name doesn't make sense. We are testing a corporate contact form, not a generic settings form...

Reply:
ok, reverted